dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -131,7 +131,7 @@ def show
       return
     end
 
-    if request.path != (canonical_path = build_script_level_path(@script_level, @extra_params))
+    if request.path != (canonical_path = build_script_level_path(@script_level, @extra_params)) && params[:view] != 'summary'
```

Comment:
I checked if Maddie had any context on this, since she was the last to modify it - but as she pointed out, this code is actually unchanged from the [very first code commit](https://github.com/code-dot-org/code-dot-org/commit/7564066231e74aaeee763becfb3af70a84ce5dc5) in the repo. i did some manual testing with paths that trigger this (e.g. localhost-studio.code.org:3000/s/coursef-2020/lessons/2/levels/2/page/0), and it seems to be fine.